lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -31,6 +31,9 @@ export default function FilePickerApp({
     authToken,
     authUrl,
     courseId,
+    // It is assumed that enableLmsFilePicker is only used by LMS apps
+    // for which we do not have an integration with the file storage.
```

Comment:
This should say "for which we *do* have an integration" surely? I'd also clarify that "the file storage" means "the LMS's file storage API".